### ISSUE TITLE
certificates in "files" are incomplete

### ISSUE THREAD

**Issue by Justin-DynamicD** [nielsengelen/val-puppet#2]
In the install.pp a repo is created referencing two gpg keys,  when testing they have unique IDs.  However, the installer only installs 1 key.  Both keys should be present in the file and installed to allow veeam to download properly.

**Comment by nielsengelen** [comment]
Could you clarify a bit more? The GPG key being added is to reference the Veeam repo and afterwards uses the GPG files via HTTP. Does the install fail or do you get an error?

**Comment by Justin-DynamicD** [comment]
Yes, the download fails claiming mismatched keys.  

In the Centos install you are copying the "RPM-GPG-KEY-VeeamSoftwareRepo" in the files directory into /etc/pki/rpm-gpg/VeeamSoftwareRepo (note the name change).   Your repo is then setup with GPG validation to that key.  Now your Repo points to "http://repository.veeam.com/keys/VeeamSoftwareRepo" and if I wget that and run "gpg --keyid-format 0xLONG /etc/pki/rpm-gpg/VeeamSoftwareRepo" I do not get the same ID between the two keys.

What it looks like is RPM-GPG-KEY-VeeamSoftware should be copied and retain it's name, and a 2nd key also get copied with a matching ID for VeeamSoftwareRepo.  With that modification I was able to download and kick off the install as expected.

Basically I put both keys in the files ... which might be overkill, but as is there's a key mismatch due to validation being enabled.

**Comment by nielsengelen** [comment]
Issue has been resolved combined with an updated chance to manage epel repo's. See pull request #3
